kubernetes/services/rabbitmq.yml.j2

```diff
@@ -0,0 +1,76 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rabbitmq
+spec:
+  replicas:             1
```

Comment:
Please add a comment explaining why it's set to 1:

RabbitMQ is stateful so we can't simply spin up several replicas without ending up with separate broker instances with separate queues. For now this has to be hard-coded to 1.

Reply:
Done.